src/status_im/transport/utils.cljs

```diff
@@ -19,8 +19,10 @@
 
 (defn message-id
   "Get a message-id"
-  [message]
-  (sha3 (pr-str message)))
+  [{:keys [from chat-id clock-value] :as m}]
```

Comment:
I don't think we should change message-ids like this considering the consequences we had to deal with the last time we did it. If we do the last time it was discussed it seemed that a form of UUID was the most reasonable choice over any kind of computed id

Reply:
Well, that's nice but if it's not deterministic we will have to maintain deduplication stuff. So if that's an issue we still can add UUID later and maintain `old-message-id` or something for a transition period. If introduced way to compute `message-id` is not worse than the one we had before this PR, it should be good enough. Other stuff is a bit out of the scope.